src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
+		return;
```

Comment:
I also opened https://github.com/erming/shout/issues/471 on the main project to keep an eye on this project-wide.